### PR TITLE
German translation update (native speaker)

### PR DESCRIPTION
**Pull request by jmfrenzel**
Fixes #issuenumber ; refs #issuenumber

Present tense short summary (50 characters or less)

More detailed description, if necessary. Try to be as descriptive as you can: even if you think that the PR content is obvious, it may not be obvious to others. Include tracebacks if helpful, and be sure to call out any bits of the PR that may be work-in-progress.

Description can have multiple paragraphs and you can use code examples inside:

``` ruby
class PostsController
  def index
    respond_with Post.limit(10)
  end
end
```

Changes proposed in this pull request:
*
*
*

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: config/locales/hyrax.de.yml [hunk 1]
```diff
@@ -168,8 +168,8 @@ de:
             success: Das Design wurde erfolgreich aktualisiert
       features:
         index:
-          action: Aktion
-          description: Beschreibung
+          feature: Features
```
Comment (mjgiarlo) [inline config/locales/hyrax.de.yml]: Did you mean to replace the `action` and `desciption` keys with duplicate `feature` and `header` keys?

Comment (jmfrenzel) [reply config/locales/hyrax.de.yml]: oops, s.t. wet wrong. You are right  action and desciption keys should not change. 
 how to handle this. New PR?
  169       features:
  170         index:
  171           action: Aktion
  172           description: Beschreibung
  173           feature: Features
  174           header: Features

File: config/locales/hyrax.de.yml [hunk 2]
```diff
@@ -170,8 +170,8 @@ de:
         index:
           action: Aktion
           description: Beschreibung
-          feature: Charakteristisch
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Keep, do not remove!

File: config/locales/hyrax.de.yml [hunk 3]
```diff
@@ -170,8 +170,8 @@ de:
...
-          header: Charakteristik
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Keep, do not remove!

File: config/locales/hyrax.de.yml (hunk 2 above)
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Ok that's it

**Comment by jmfrenzel** [comment]
Hope, I handled it correctly. This part of Github was new to me,

**Comment by mjgiarlo** [comment]
Hi, @jmfrenzel. Just so I understand, the `feature` and `header` translations should be set to `Features`? That's idiomatic in German?

**Comment by jmfrenzel** [comment]
In terms of software it is [official](https://www.duden.de/rechtschreibung/Feature#Bedeutung3).

**Comment by mjgiarlo** [comment]
Thank you, @jmfrenzel!

**Pull request merged**